### PR TITLE
add BA index

### PR DESCRIPTION
**Pull request by adrpadua**
Add blood angels index + boilerplate outlined in root readme

### CODE REVIEW

**Comment by korzxd** [comment]
Hey, thank you for the contribution.

I've had a quick browse and this seems great. I'll review this properly over the weekend and get it added. 👍

**Comment by adrpadua** [comment]
anytime :) thanks for putting everything together here

**Review by korzxd — COMMENTED** [review]
First off thanks for all your hard work i do appreciate the help to this repository, the added section of outlining modifiers to use for stratagems is a great idea and i will most likely follow suit with the other Factions.

All the unit characteristics and near all of the weapon profiles are correct.
There are a few things outlined in this review that will need to be addressed first, most of them being just minor Keywords and missing abilities.

I also haven't reviewed the Leader attached squads yet and you may have to double check those.

Aside from that, brilliant work! Once these have been addressed I'll merge the request. 👍

File: Space Marines/Blood Angels/Readme.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  * **`Lord Regent of the Imperium Nihilus:`** <br> Enable the profile ability `+1 to hit roll (if attacker)`.
+
+* #### Gabriel Seth:
+  * **`Whirlwind of Gore`** <br> Enable the profile ability `Increase attacks by #` and adjust the value.
+
+* #### Lemartes:
+  * **`Guardian of the Lost`** <br> Enable the profile ability `Damage is greater than 1 : -1 to damage (if defender)`.
+  * **`Fury Unbound`** <br> Add `LETHAL HITS` to the unit profile.
+
+* #### Sanguiary Priest:
```
[the remark this answers is not in the document]
Comment (adrpadua) [reply Space Marines/Blood Angels/Readme.md]: This one's not a psyker I believe

[the remark this answers is not in the document]
Comment (adrpadua) [reply Space Marines/Blood Angels/Readme.md]: Added the CHARACTER keyword, I think that's what you meant

File: Space Marines/Blood Angels/Readme.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+## Infantry
+
+* #### Death Company Intercessors:
+  * **`Black Rage`** <br> Enable the profile ability `Re-roll all possible failed hit rolls (if attacker)`.
+
+* #### Death Company Marines:
+  * **`Black Rage`** <br> Enable the profile ability `Re-roll all possible failed hit rolls (if attacker)`.
+  * **`An Honourable Death in Combat`** <br> Add `SUSTAINED HITS 1`, `SUSTAINED HITS 2` to the unit profile and modify model/weapon count accordingly.
+
+* #### Death Company Marines with Jump Pack:
```
[the remark this answers is not in the document]
Comment (adrpadua) [reply Space Marines/Blood Angels/Readme.md:74]: Added 'Black Rage' profile ability as well

**Comment by adrpadua** [comment]
Thanks for the thorough review :) Pushed a new commit with all the feedback implemented

**Pull request merged**